web-app/src/smart-locks/components/smart-lock-list-item.component.vue

```diff
@@ -0,0 +1,34 @@
+<script setup lang="ts">
+import DoorsIcon from "../../shared/icons/doors-icon.component.vue";
+
+const props = defineProps<{
+  name: string;
+  keysCount: number;
+  id: number;
+}>();
+
+const emit = defineEmits<{
+  (e: "manage", id: number): void;
+}>();
+
+const onClick = () => {
+  emit("manage", props.id);
+};
```

Comment:
If this component uses a `button` as template root, this is not needed because whoever uses this component will simply have to use the built-in `@click` instead of a custom name